CONTRIBUTING.md

```diff
@@ -65,5 +69,8 @@ We have also automated the process via [clang-format](http://clang.llvm.org/docs
 
 [How to clang-format a diff](http://clang.llvm.org/docs/ClangFormat.html#script-for-patch-reformatting)
 
+## Keeping up to date with the master branch
```

Comment:
can you carry this section up to submitting a patch and make it a subheader with `###`. it breaks the narrative when you have coding conventions in the middle